### ISSUE TITLE
Compiler: Choking on expander.expandModules (Windows)

### ISSUE THREAD

**Issue by am11** [mozilla/sweet.js#386]
Hello!

One of the [sweet.js test cases](https://github.com/madskristensen/WebEssentials2013/blob/48d237e963db8a3434daf377a8d043a6e330046e/WebEssentialsTests/Tests/NodeExecutors/SweetJsCompilationTests.cs#L26-L41) in Web Essentials is failing. WE consumes sweet.js npm as a web-service ([srv-sweetjs](https://github.com/am11/WebEssentials2013/blob/96ff6993fddd5d9550e43f6d830d6d45bb9911f9/EditorExtensions/Resources/server/services/srv-sweetjs.js)) to WE's web-server.

Looks like the crash is caused by `expander.expandModule` at [sweet.js: Line 73](https://github.com/mozilla/sweet.js/blob/8b705a015685769e12030f7bcfe772d1f0898ec5/lib/sweet.js#L73).

I isolated the reproduction steps by running the following in node.js shell:

```cmd
> require("sweet.js").compile("var foo = 100;", {filename: "C:\\temp\\test.sjs", sourceMap: true})
TypeError: Object #<Object> has no method 'expose'
    at eval (eval at <anonymous> (C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node_modules\sweet.js\lib\scopedEval.js:16:69), <anonymous>:93:75)
    at expandMacro (C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node_modules\sweet.js\lib\expander.js:1223:18)
    at step (C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node_modules\sweet.js
\lib\expander.js:1613:30)
    at enforest (C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node_modules\sweet.js\lib\expander.js:1751:16)
    at step (C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node_modules\sweet.js\lib\expander.js:1558:42)
    at step (C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node_modules\sweet.js\lib\expander.js:1703:28)
    at enforest (C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node_modules\sweet.js\lib\expander.js:1751:16)
    at expandToTermTree (C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node_modules\sweet.js\lib\expander.js:2009:17)
    at expandTermTreeToFinal (C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node
_modules\sweet.js\lib\expander.js:2285:34)
    at C:\Users\Adeel\Source\Repos\WebEssentials2013\EditorExtensions\Resources\nodejs\tools\node_modules\sweet.js\lib\expander.js:2202:24
```

Even using the CLI throws the same error on Windows (**same woks fine on Ubuntu**).

Also, the exception variable throws the same error when I try to `console.log(error)` like [this](https://github.com/am11/WebEssentials2013/blob/96ff6993fddd5d9550e43f6d830d6d45bb9911f9/EditorExtensions/Resources/server/services/srv-sweetjs.js#L56). Apparently, the object is unreadable.

**Comment by disnet** [comment]
I'm mass closing issues due to the release of 1.0 obviating lots of bugs. If you feel this issue is something we still need to address feel free to reopen.

**Comment by am11** [comment]
I can confirm that this issue is fixed when tested with version 1.0.1.
